README.md

```diff
@@ -128,6 +128,29 @@ if (!supportsTime) {
 
 This will add a global `TimePolyfill` function to the page.
 
+## Shadow DOM
+
+When your code is inside a component that resides in the Shadow DOM, the polyfill will not be able to find your label
+element. For this case, you can pass your label element in directly.
+
+```
+<label id="myLabel"></label>
+<input type="time" id="timeInput">
+```         
+
+```               
+import * as timePolyFill from 'time-input-polyfill';
```

Comment:
For normal JavaScript users, the import is just this:

```
import timePolyfill from 'time-input-polyfill'
```

The * thing is something only Typescript users have to do.

Reply:
If you want to add a mention of * then it can go up in the area of the readme where I say how to import.